mbq/client/contrib/permissions.py

```diff
@@ -290,9 +290,18 @@ def _has_permission(
 
     def has_global_permission(self, person_id: UUIDType, scope: str) -> bool:
         """Test whether the scope is granted to the person on the global scope."""
-        with self.collector.timed("has_global_permission.time"):
+        with self.collector.timed(
```

Comment:
how does this pattern work? what's the interaction between the context manager and then the increment call below? I presume that the context manager times the block inside of it and then the call below emits the metric, but does the timing just get passed to the emission automagically?